src/compile/nodes/Element.ts

```diff
@@ -1159,3 +1169,9 @@ const events = [
 			name === 'volume'
 	}
 ];
+
+function camelCase(str) {
```

Comment:
I don't know how I feel about camel casing the class name to get the state field name. On the one hand, I can see how this would let you follow CSS class naming conventions more easily - but on the other, we're not doing this transformation anywhere else (on component prop names, etc.). I think I'm leaning towards saying that the class name and property name have to exactly match.